issue-001/solution.js

```diff
@@ -1,3 +1,19 @@
 function dataTypeCheck(a) {
-    // implement your solution here
-}
\ No newline at end of file
+
+    if (a === +a && a % 1 === 0) {
```

Comment:
This was a brilliant way of checking for `int` vs `float` in JS, never thought of that. Good stuff 👍